### PR TITLE
Add fix for clang install of non m1 mac

### PR DESCRIPTION
**Pull request by milescattini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,5 +105,10 @@ To install a C++ compiler on Windows 10/11, follow these steps:
 3. Download the MinGW installer from the [MinGW website](https://sourceforge.net/projects/mingw/).
 4. Run the installer and select the `gcc` component.
 
+## Mac Running Intel 
+When running a Mac with inteel hardware (not M1), you may run into _clang: error: the clang compiler does not support '-march=native'_ during pip install.
```
Comment (R-Y-M-R) [inline README.md:109]: Small typo

**Review by R-Y-M-R — CHANGES_REQUESTED** [review]
Really good contribution for Mac users! Minor typo for you to fix. Thanks for the contribution!

File: README.md (hunk 1 above)
Comment (milescattini) [reply README.md]: Ha! Whoops. Fixed.

**Review by imartinez — APPROVED** [review]
Thanks for this contribution helping the community!

**Pull request merged**